src/development/tools/devtools/performance.md

```diff
@@ -108,6 +118,43 @@ with the mouse wheel / trackpad
 You can click an event to view CPU profiling information in the CPU profiler
 below, described in the next section.
 
+## Enhance tracing 
+
+To view more detailed tracing in the timeline events chart, use the
+options in the enhance tracing dropdown:
+
+{{site.alert.note}}
+Frame times may be negatively affected when these options are enabled.
+{{site.alert.end}}
+
+![Screenshot of enhance tracing dropdown]({{site.url}}/assets/images/docs/tools/devtools/enhance-tracing.png)
+
+To see the new timeline events, trigger a new frame by interacting with
+your app.
+
+### Track widget builds
+
+To see the build() methods in the timeline, enable the Track Widget
```

Comment:
r/To see the build() methods in the timeline/To see widget builds in the timeline